packages/header/src/HeaderBrand.tsx

```diff
@@ -48,7 +48,7 @@ export function HeaderBrand({
 			{logo ? <Box borderRight display={{ xs: 'none', md: 'block' }} /> : null}
 			<Stack justifyContent="center">
 				<Text
```

Comment:
Does this need an `as="h2"` - not sure what the right a11y choice is here.

Reply:
No in this case we elected to not use Headings, due to findings in a recent a11y review. We want the page title (h1) to be the first heading read out.